app/src/ui/dispatcher/dispatcher.ts

```diff
@@ -204,6 +204,11 @@ export class Dispatcher {
     return this.appStore._selectRepository(repository)
   }
 
+  /** Load the working directory status. */
+  public loadStatus(repository: Repository): Promise<boolean> {
+    return this.appStore._loadStatus(repository)
```

Comment:
I was able to remove the public API because inside `Dispatcher` we just invoke `_loadStatus` directly.

This is my preferred solution, to be consistent with other callers:

```diff
diff --git a/app/src/ui/dispatcher/dispatcher.ts b/app/src/ui/dispatcher/dispatcher.ts
index 1ccb8a226..4173ae3d8 100644
--- a/app/src/ui/dispatcher/dispatcher.ts
+++ b/app/src/ui/dispatcher/dispatcher.ts
@@ -1510,7 +1510,7 @@ export class Dispatcher {
       forceWithLease: true,
     })
 
-    await this.loadStatus(repository)
+    await this.appStore._loadStatus(repository)
   }
 
   public setConfirmForcePushSetting(value: boolean) {
```